src/pages/dmt5-trading-signals/index.js

```diff
@@ -102,6 +102,10 @@ const Separator = styled.div`
 
 const DMT5TradingSignals = () => {
     const [active_tab, setActiveTab] = useTabState(['signal-subscriber', 'signal-provider'])
+    const [not_first_loading, setNotFirstLoading] = useState(false) //needs to fix bug with hightlight of the 1st loading
```

Comment:
```suggestion
    const [is_mounted, setMounted] = useState(false) // needed to fix tab highlighting not being rerendered during first load
```

Reply:
A suggestion for renaming the variable to indicate that it is called once the whole component has mounted.